spec/controllers/hyrax/single_use_links_viewer_controller_spec.rb

```diff
@@ -60,5 +60,19 @@
         expect(response).to render_template("hyrax/single_use_links_viewer/single_use_error", "layouts/error")
       end
     end
+
+    describe "#current_ability" do
+      context "when the key is not found" do
```

Comment:
Everything looks good. Could you explain the context around this test? 

Was it added to test something related to current_ability but not the changes introduced in this ticket?

Reply:
Asking about the "when the key is not found" test, the highlighting is kind of awkward.

Reply:
The "key is not found" part refers to the `before` block just below this comment, which makes it so the single use link key does not exist when the tests run. That state is what causes the bug being addressed in the ticket, where any attempts to check the user's permissions when the SingleUseLink they are trying to access doesn't exist causes the server to return a 500 response. The test is to demonstrate that `current_ability` doesn't blow up anymore after this change. I borrowed the "when the key is not found" text/logic from other tests in this spec file.

Reply:
Got it, thanks!